qtconsole/ansi_code_processor.py

```diff
@@ -185,6 +182,20 @@ def set_csi_code(self, command, params=[]):
             count = params[0] if params else 1
             self.actions.append(ScrollAction('scroll', dir, 'line', count))
 
+        elif (command == 'A'):  # Move N lines Up
```

Comment:
```suggestion
        elif command == 'A':  # Move N lines Up
```

No need for using parenthesis here